cpp/oneapi/dal/test/engine/dataframe_common.hpp

```diff
@@ -259,6 +259,7 @@ class dataframe_builder {
             : impl_(new dataframe_builder_impl{ row_count, column_count }) {}
 
     dataframe_builder& fill_uniform(double a, double b, std::int64_t seed = 7777);
+    dataframe_builder& fill_normal(double a, double b, std::int64_t seed = 7777);
```

Comment:
Use more appropriate var names for normal distribution, e.g., center/loc and std